### ISSUE TITLE
does Singularity support multi-job dependency?

### ISSUE THREAD

**Issue by junneyang** [HubSpot/Singularity#1007]
thanks!

**Comment by ssalinas** [comment]
No, currently there is no support for this. However, there are webhooks that will fire when tasks finish that you could subscribe to to have another service trigger a new job, or the previous job could hit the api to trigger the next job.

I will keep this open for now. Job dependency is something that has come up a few times as a feature request

**Comment by jeffreyksmithjr** [comment]
Just noting that we also have this need. Any ideas on how best to use another tool to integrate with Singularity to solve this?

**Comment by alcuma** [comment]
Hi, do you have any news about this enhancement? It would be great to have this feature.

Thanks

**Comment by ssalinas** [comment]
a note that another requested/related feature here (#1419) is dependencies between services/workers in which one must be launched before another
